app/controllers/admin/books_controller.rb

```diff
@@ -0,0 +1,79 @@
+class Admin::BooksController < ApplicationController
+  before_action :load_book, except: %i(index create new)
+  before_action :load_books, only: %i(index create update destroy)
+
+  def index; end
+
+  def show; end
+
+  def delete; end
+
+  def new
+    @book = Book.new
+  end
+
+  def create
+    if @book = Book.create(book_params)
+      respond_to do |format|
```

Comment:
Viết lại như này cho đỡ rối nha em, rồi tách cái cuối ra dùng chung toàn bộ
if @book = Book.create(book_params)
flash[:success] = t(".msg-create")
else
flash[:success] = t(".msg-error")
end
respond_to do |format|
format.js {}
end